### PR TITLE
f-icons@4.5.0 - Add new cash small icon

### PR DESCRIPTION
**Pull request by jamieomaguire**
v4.5.0
------------------------------
*January 17 2022*

### Added
- Cash small icon

### CODE REVIEW

**Review by matthewhardern — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 
+v4.4.0
```
Comment (simonsandersje) [inline CHANGELOG.md:7]: Do you mean 4.5.0?

Comment (jamieomaguire) [reply CHANGELOG.md:7]: I totally did

**Review by matthewhardern — APPROVED** [review]

**Review by simonsandersje — APPROVED** [review]

File: icons/cash-small.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0.22 0.97 13.56 9.19">
```
Comment (DamianMullins) [inline icons/cash-small.svg:1]: Did this icon come from the PIE iconset? The [only one I could find there](https://www.figma.com/file/tGiSNOWHbX9DD1KrWwlptG/PIE-Icons?node-id=96%3A247) looks a little different to this one?

Comment (jamieomaguire) [reply icons/cash-small.svg:1]: Hey @DamianMullins - got it from this resource that I was told was the reference for all new icon sets: 

https://www.figma.com/file/k7gPJ4MZRUj4nlZK2hL0Op/%E2%9A%A1%EF%B8%8FPIE-2.0-Icons?node-id=588%3A58 

There's a number of variants for the cash icon

Comment (DamianMullins) [reply icons/cash-small.svg:1]: Awesome, my bad, I was looking at the old icons!

**Pull request merged**